### ISSUE TITLE
xml from AMS reflects transcript status

### ISSUE THREAD

**Issue by sroosa** [WGBH/AAPB2#1188]
Part of #1178 

We are going to use an annotation in the xml to reflect:
- whether the website should look for a transcript on S3
- whether the website should display the transcript (is it an un-cleaned up one we're only using for indexing?)
- whether the transcript should be displayed on On Location or in the Online Reading Room

The annotation will be a pbcoreAnnotation with @annotationType = Transcript Status. The possible values for this element will be "Indexing Only Transcript" "On Location Transcript" and "Online Reading Room Transcript"

We cannot rely on the Level of User Access annotation to determine the On Location/Online Reading Room distinction because we will be showing transcripts for something that are not even digitized and so they won't have a Level of User Access annotation at all. 

Done When: The xml exported from the AMS for records with transcripts include the annotation mentioned above.

**Comment by afred** [comment]
Question: should this effort include all of the effort of putting the actual metadata in AMS? Cause we don't know how long that will take.

**Comment by caseyedavis12** [comment]
@afred: no, just estimate the amount of time it will take for developers to implement this on the website after the metadata is put into the AMS.

**Comment by sroosa** [comment]
@jasoncorum: Can you confirm for me that the transcript annotation made it into the solr index as expected? Thanks!

**Comment by jasoncorum** [comment]
@sroosa - Once I saw how we implemented captions, I realized that this wouldn't be needed for this portion of the implementation. The way we handle captions and, now, transcripts is that on ingest we check s3 for any transcript, if there's a transcript we add it to an indexed field. 

We'll end up using this once we need to determine whether to display the transcript or not. So, maybe we put this back in the backlog and tie it to #1178?
